app-shells/bash/bash-5.1.recipe

```diff
@@ -0,0 +1,87 @@
+SUMMARY="The GNU Bourne Again Shell"
+DESCRIPTION="Bash is an sh-compatible command language interpreter that \
+executes commands read from the standard input or from a file. Bash also \
+incorporates useful features from the Korn and C shells (ksh and csh)."
+HOMEPAGE="https://www.gnu.org/software/bash/"
+COPYRIGHT="1987-2020 Free Software Foundation, Inc."
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://ftpmirror.gnu.org/bash/bash-5.1.tar.gz"
+CHECKSUM_SHA256="cc012bc860406dcf42f64431bcd3d2fa7560c02915a601aba9cd597a39329baa"
+
+SOURCE_DIR="bash-5.1"
+PATCHES="
+	bash-kill_by_name.patch
+	bash-5.0.patchset
+	"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64 sparc m68k"
+
+PROVIDES="
+	bash = $portVersion
+	cmd:bash = $portVersion
+	cmd:bashbug = $portVersion
+	cmd:sh
+	"
+REQUIRES="
+	haiku
+	lib:libhistory
+	lib:libreadline
```

Comment:
Could you check that bash gets a dependency on 8.1? that's how bash is packaged, even if 8.1 is compatible with 8.0

Reply:
~> objdump -x /boot/system/bin/bash | grep NEEDED
  NEEDED               libreadline.so.8
  NEEDED               libhistory.so.8
  NEEDED               libncurses.so.6
  NEEDED               libintl.so.8
  NEEDED               libroot.so

Reply:
technically right, in reality bash 5.1 is to be deployed with readline 8.1: this is what is tested by bash developers.

See example in Gentoo: 
https://gitweb.gentoo.org/repo/gentoo.git/tree/app-shells/bash/bash-5.1.ebuild#n37
https://gitweb.gentoo.org/repo/gentoo.git/tree/app-shells/bash/bash-5.1.ebuild#n56